config/webpack.common.js

```diff
@@ -48,7 +48,7 @@ const METADATA = {
  * See: http://webpack.github.io/docs/configuration.html#cli
  */
 module.exports = function (options) {
-  const isProd = options.env === 'production';
+  const isProd = !!options.isProd;
```

Comment:
Why did you change this and how does this work?

Reply:
oh, I see you added it at the bottom. Still not sure why you needed this.

Reply:
I had to change it to run perf. Since per sets the environment to performance.

Maybe perf should also be production?

Reply:
yes, we should set perf to prod since it is a reflection of it

Reply:
ok let me change the environment instead of adding this option